sdk/purview/purview-account-rest/samples-dev/collections.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * This sample demonstrates how get a list of typedefs
+ *
+ * @summary gets a list of typedefs for entities
+ * @azsdk-weight 40
+ */
+
+import PurviewAccount from "@azure-rest/purview-account";
+import { DefaultAzureCredential } from "@azure/identity";
+import dotenv from "dotenv";
+
+dotenv.config();
+
+const endpoint = process.env["ENDPOINT"] || "";
+
+async function main() {
+  console.log("== List collections sample ==");
+  const client = PurviewAccount(endpoint, new DefaultAzureCredential());
+
+  const dataSources = await client.path("/collections").get();
+
+  if (dataSources.status !== "200") {
+    throw dataSources;
```

Comment:
do we want to throw `dataSources.body.error`?

Reply:
currently I have some permission problems accessing the purview account. will get back to this and update the test recordings when that's resolved. Thanks

Reply:
updated to log the error code.